mlflow/resource_model.go

```diff
@@ -0,0 +1,73 @@
+package mlflow
+
+import (
+	"context"
+
+	"github.com/databrickslabs/terraform-provider-databricks/common"
+	"github.com/databrickslabs/terraform-provider-databricks/mlflow/api"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// MLFlowModel defines the parameters that can be set in the resource.
+type Model struct {
+	Name        string    `json:"name" tf:"force_new"`
+	Tags        []api.Tag `json:"tags,omitempty" tf:"force_new"`
+	Description string    `json:"description,omitempty"`
+}
+
+// ResourceDashboard ...
+func ResourceMLFlowModel() *schema.Resource {
+	s := common.StructToSchema(
+		Model{},
+		func(m map[string]*schema.Schema) map[string]*schema.Schema {
+			return m
+		})
+
+	return common.Resource{
+		Create: func(ctx context.Context, data *schema.ResourceData, c *common.DatabricksClient) error {
```

Comment:
please make sure that ResourceData is always named `d` - `d *schema.ResourceData`. it's a convention.